main.nf

```diff
@@ -1929,6 +1934,236 @@ process UNICYCLER_QUAST {
     """
 }
 
+////////////////////////////////////////////////////
+/* --                MINIA                     -- */
+////////////////////////////////////////////////////
+
+/*
+ * STEP 6.3: De novo assembly with minia
+ */
+process MINIA {
+    tag "$sample"
+    label 'process_medium'
+    publishDir "${params.outdir}/assembly/minia", mode: params.publish_dir_mode
+
+    when:
+    !params.skip_assembly && 'minia' in assemblers
+
+    input:
+    set val(sample), val(single_end), file(reads) from ch_kraken2_minia
+    output:
+    set val(sample), val(single_end), file("*scaffolds.fa") into ch_minia_overlap_scaffolds,
+                                                                 ch_minia_quast,
+                                                                 ch_minia_blast,
+                                                                 ch_minia_abacas,
+                                                                 ch_minia_plasmidid
+
+    script:
+    if (single_end) {
+        """
+        minia -kmer-size 51 -abundance-min 20 -in $reads -out ${sample}.k51.a20

```

Comment:
Should we pull any `minia` parameters used here out to config?  Is there any utility in doing multi-k here?